boards/nxp/frdm_mcxa153/frdm_mcxa153.yaml

```
@@ -0,0 +1,19 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: frdm_mcxa153
+name: NXP FRDM MCXA153
+type: mcu
+arch: arm
+ram: 32
```

```suggestion
ram: 24
```
The `zephyr,sram` node points to `sram0`, which is 24 KB.